metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -184,3 +187,110 @@ Feature: Transitive release date control for child objects of a Study
+    And set the URL to TAXONOMY
+    Then the response code should be 201
+
+    When I request POST /reference-sequences with JSON-like payload:
+    """
+          "name": "GRCh37",
+          "patch": "p2",
+          "accession": "GCA_000001407.3",
+          "type": "GENOME_ASSEMBLY",
+          "taxonomy": "TAXONOMY"
+    """
+    And set the URL to REFERENCE_SEQUENCE
+    Then the response code should be 201
+
+    When I create a study with unreleasedStudy for accession
+    And set the URL to STUDY
+    Then the response code should be 201
+
+    When I create a study with unreleasedStudy1 for accession
+    And set the URL to STUDY1
+    Then the response code should be 201
+
+    When I create an analysis with Analysis for accession, REFERENCE_SEQUENCE for reference sequence and STUDY for study
+    And set the URL to ANALYSIS
+    Then the response code should be 201
+
+    When I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "File",
+        "version": 1
+        },
+      "hash": "Hash3",
+      "name": "File3",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    And set the URL to FILE
+    Then the response code should be 201
+
+    When I request PATCH ANALYSIS with list FILE of files
+    Then the response code should be 2xx
+
+    When I request POST /publications with JSON payload:
+    """
+    {
+      "publicationId": "publication1"
+    }
+    """
+    Then set the URL to PUBLICATION
+    And the response code should be 201
+    When I request PATCH STUDY with list PUBLICATION of publications
+    Then the response code should be 2xx
+
+    When I request POST /webResources with JSON payload:
+    """
+    {
+      "type": "CENTER_WEB",
+      "resourceUrl": "http://test-release-date.example.com"
+    }
+    """
+    And set the URL to WEBRESOURCE
+    Then the response code should be 201
+
+    When I request POST /webResources with JSON payload:
+    """
+    {
+      "type": "CENTER_WEB1",
+      "resourceUrl": "http://test-release-date1.example.com"
+    }
+    """
+    And set the URL to WEBRESOURCE1
+    Then the response code should be 201
+
+    When I request PATCH STUDY with list WEBRESOURCE of resources
+    Then the response code should be 2xx
+    When I request PATCH STUDY1 with list WEBRESOURCE of resources
+    Then the response code should be 2xx
+    When I request PATCH STUDY1 with list WEBRESOURCE1 of resources
+    Then the response code should be 2xx
+
+    When I request PATCH STUDY with patch and day tomorrow
+    Then the response code should be 2xx
+    When I request PATCH STUDY1 with patch and day tomorrow
+    Then the response code should be 2xx
+
+    When I set authorization with testuser having default role
+    And I request GET with value of STUDY
+    Then the response code should be 200
+    And I request GET with value of STUDY1
+    Then the response code should be 4xx
```

Comment:
It would be good to put exact HTTP code, especially in fail cases. Fox ex, response here could have been 404 or 401, for which the expectation can not be same.